modules/em/test/test_bayesem3d_derivatives.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import print_function
+import IMP
+
+import IMP.em
+import IMP.core
+import IMP.atom
+import IMP.test
+
+import random
+import math
+import sys
+
+IMP.set_log_level(IMP.SILENT)
+IMP.set_check_level(IMP.NONE)
```

Comment:
Don't set IMP stuff at the global level, since this will be done at import time (and potentially affect all tests run in this process). If you must change the log level, do so in your test setup method or the test itself.